scico/linop/xray/_xray.py

```diff
@@ -193,3 +194,116 @@ def _calc_weights(x0, dx, nx, angle, y0):
     weights = jnp.minimum(distance_to_next, width) / width
 
     return inds, weights
+
+
+class Parallel3dProjector:
+    """General-purpose, 3D, parallel ray X-ray projector."""
+
+    def __init__(
+        self,
+        input_shape: Shape,
+        P: ArrayLike,
```

Comment:
A bit of tension here between mathematical convention and coding standards, but my inclination would be to make this lower case. Perhaps "proj_matrices" or "projections"?

Reply:
(Potentially optional, consider)